src/Validator/Multiple.php

```diff
@@ -18,25 +18,28 @@ class Multiple extends Validator
      */
     protected $rules = [];
 
+    protected $type = self::TYPE_MIXED;
+
     /**
      * Constructor
      *
      * Multiple constructor can get any number of arguments containing Validator instances using PHP func_get_args function.
      *
      * Example:
      *
-     * $multiple = new Multiple($validator1, $validator2, $validator3);
+     * $multiple = new Multiple([$validator1, $validator2]);
+     * $multiple = new Multiple([$validator1, $validator2, $validator3], SELF::TYPE_STRING);
      */
-    public function __construct()
+    public function __construct(array $rules, ?string $type = null)
```

Comment:
Can we do this instead? not sure if the syntax is supported

```php
    public function __construct(array $rules, ?string $type = self::TYPE_MIXED)
```

If yes, we can drop the `if null` check

Reply:
Doesn't work